test/powershell/Modules/Microsoft.PowerShell.Utility/XMLCommand.Tests.ps1

```diff
@@ -49,4 +49,111 @@ Describe "XmlCommand DRT basic functionality Tests" -Tags "CI" {
 		$results.Property1 | Should Be $property1
 		$results.Property2 | Should Be $property2
     }
+
+	It "Export-Clixml StopProcessing should succeed" {
+		$ps = [PowerShell]::Create()
+		$ps.AddCommand("Get-Process")
+		$ps.AddCommand("Export-CliXml")
+		$ps.AddParameter("Path", $testfile)
+		$ps.BeginInvoke()
+		$ps.Stop()
+		$ps.InvocationStateInfo.State | Should Be "Stopped"
+	}
+
+	It "Import-Clixml StopProcessing should succeed" {
+		1,2,3 | Export-Clixml -Path $testfile
+		$ps = [PowerShell]::Create()
+		$ps.AddCommand("Get-Process")
+		$ps.AddCommand("Import-CliXml")
+		$ps.AddParameter("Path", $testfile)
+		$ps.BeginInvoke()
```

Comment:
I can create a larger clixml file to import so that it's more likely StopProcessing() is called during operation and it hasn't completed.  Also noticed `Get-Process` shouldn't be there.